src/community/pom.xml

```diff
@@ -427,6 +429,13 @@
         <modules>
           <module>vectortiles</module>
         </modules>
+    </profile>    
+    <profile>
+      <id>jdbcstore</id>
```

Comment:
@NielsCharlier the ```<id>``` and ```<modules>``` tags are peers and should have the same indentation. Some other profiles in this pom have the same problem (did you cut and paste?) but you only need to fix the one you are adding.  :-)